notebook/static/notebook/js/actions.js

```diff
@@ -545,13 +552,13 @@ define(function(require){
         },
     };
 
-    // private stuff that prepend `.ipython` to actions names
+    // private stuff that prepend `jupyter-notebook:` to actions names
     // and uniformize/fill in missing pieces in of an action.
     var _prepare_handler = function(registry, subkey, source){
-        registry['ipython.'+subkey] = {};
-        registry['ipython.'+subkey].help = source[subkey].help||subkey.replace(/-/g,' ');
-        registry['ipython.'+subkey].help_index = source[subkey].help_index;
-        registry['ipython.'+subkey].icon = source[subkey].icon;
+        registry['jupyter-notebook:'+subkey] = {};
+        registry['jupyter-notebook:'+subkey].help = source[subkey].help||subkey.replace(/-/g,' ');
+        registry['jupyter-notebook:'+subkey].help_index = source[subkey].help_index;
+        registry['jupyter-notebook:'+subkey].icon = source[subkey].icon;
```

Comment:
Yeah, you make a good point about how the names are described in the UI. I
did change it slightly in the UI as well to better reflect it - but still
not pretty...

On Mon, Oct 12, 2015 at 9:05 PM, Matthias Bussonnier <
notifications@github.com> wrote:

> In notebook/static/notebook/js/actions.js
> <https://github.com/jupyter/notebook/pull/598#discussion_r41817844>:
>
> >      // and uniformize/fill in missing pieces in of an action.
> >      var _prepare_handler = function(registry, subkey, source){
> > -        registry['ipython.'+subkey] = {};
> > -        registry['ipython.'+subkey].help = source[subkey].help||subkey.replace(/-/g,' ');
> > -        registry['ipython.'+subkey].help_index = source[subkey].help_index;
> > -        registry['ipython.'+subkey].icon = source[subkey].icon;
> > +        registry['jupyter-notebook:'+subkey] = {};
> > +        registry['jupyter-notebook:'+subkey].help = source[subkey].help||subkey.replace(/-/g,' ');
> > +        registry['jupyter-notebook:'+subkey].help_index = source[subkey].help_index;
> > +        registry['jupyter-notebook:'+subkey].icon = source[subkey].icon;
>
> Ok, it's just that the name of the category will show up in the CP, and it
> feel weird to actually in UI, and it feel weird to precise that this apply
> to "jupyter notebook"
> [image: screen shot 2015-10-12 at 18 01 55]
> <https://cloud.githubusercontent.com/assets/335567/10442839/5e28140c-710b-11e5-949c-6369eb921b3f.png>.
>
>
> Feel a bit like Truman show <https://youtu.be/Inscky6EyQ8?t=1m31s>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/notebook/pull/598/files#r41817844>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com